site/content/docs/main/restore-resource-modifiers.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Restore Resource Modifiers"
+layout: docs
+---
+
+## Resource Modifiers
+Velero provides a generic ability to modify the resources during restore by specifying json patches. The json patches are applied to the resources before they are restored. The json patches are specified in a configmap and the configmap is referenced in the restore command. 
+
+**Creating resource Modifiers**
+
+Below is the two-step of using resource modifiers to modify the resources during restore.
+1. Creating resource modifiers configmap
+
+   You need to create one configmap in Velero install namespace from a YAML file that defined resource modifiers. The creating command would be like the below:
+   ```bash
+   kubectl create cm <configmap-name> --from-file <yaml-file> -n velero
+   ```
+2. Creating a restore reference to the defined resource policies
+
+   You can create a restore with the flag `--resource-modifier-configmap`, which will reference the current restore to the defined resource modifiers. The creating command would be like the below:
```

Comment:
I would replace "which will reference the current restore to the defined resource modifiers" with "which will apply the defined resource modifiers to the current restore".

Reply:
Done.